### DISCUSSION TITLE
CDHSample.plotPredictorPerformanceHeatmap() issue

### DISCUSSION THREAD

**Discussion by jonathangebru** [pegasystems/pega-datascientist-tools#146] [Q&A] [answered]
Another issue I want to highlight is the calling of CDHSample.plotPredictorPerformanceHeatmap(), for which I get the following AttributeError:  module 'numpy' has no attribute 'bool'.
`np.bool` was a deprecated alias for the builtin `bool`. To avoid this error in existing code, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:  https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations

**Comment by StijnKas** [comment, marked as answer]
Hi @jonathangebru, thanks for bringing this up. I'm a little confused though, because I don't think we use `np.bool` anywhere in our code. Are you on the latest version of plotly, pandas and polars? Perhaps try updating those individual libraries and/or running `pip install --upgrade pdstools` to get to the latest pdstools version as well. This feels like a mismatch of versions, where you may be on a newer version of numpy, but an older plotly version. 

If this is indeed not a dependency issue but somewhere in our code, could you please open up an issue including the full stacktrace of the error? Then we can check where this originates from and fix it. Thanks!

**Comment by jonathangebru** [comment]
Hi Stijn, 

You're right! I upgraded my plotly version and this worked!
